### ISSUE TITLE
Delay exchange rate updates

### ISSUE THREAD

**Issue by gregdhill** [interlay/interbtc-spec#58]
With the new medianizer, exchange rate updates are always processed on the next block and enforced immediately. I believe Maker delays this update to allow participants time to respond.

**Comment by nud3l** [comment]
Yes, Maker has a delay between exchange rates being submitted and when they go live, see here: https://makerdao.world/en/faqs/oracles/

We should adopt something like this as well to allow vaults to react to incoming exchange rate updates in the future as well.
